### ISSUE TITLE
fix: do not commit binary files during --apply

### ISSUE THREAD

**Issue by YodaEmbedding** [Anty0/updaurpkg#4]
`updaurpkg --apply` should only add `PKGBUILD` and `.SRCINFO` files... and nothing further!

Currently, it adds `.tar.zst` and `.tar.gz` files, too.

**Comment by Anty0** [comment]
Yeah, I use `.gitignore` to avoid that - I want to be able to commit changes in other files in case the version update needs some changes outside of the `PKGBUILD` and `.SRCINFO`. I usually add files I don't want to commit to `.gitignore`.

**Comment by Anty0** [comment]
But there might be a better way - for example it would be possible to commit other files while specifically ignoring downloaded/built files.
